BlitzBoard/Features/HomeScreenViewController.swift

```diff
@@ -1,16 +1,66 @@
+// HomeScreenViewController.swift
+// BlitzBoard
 //
-//  HomeScreenViewController.swift
-//  BlitzBoard
-//
-//  Created by Kiasha Rangasamy on 2024/05/06.
+// Created by Kiasha Rangasamy on 2024/04/11.
 //
 
 import UIKit
 
-class HomeScreenViewController: UIViewController {
-
+class HomeScreenViewController: UIViewController, ViewModelDelegate {
+    
+    func reloadView() {
+    }
```

Comment:
Please move these functions to the bottom of the file in an extension 
Example 
extension HomeScreenViewController: ViewModelDelegate { 
 func reloadView() 
}